docs/synapse/userguides/storm_adv_methods.rstorm

```diff
@@ -7,28 +7,47 @@
+  In the examples below, the ``$lib.print()`` library function is used to display the value returned
+  when a specific built-in variable or method is called. This is done for illustrative purposes only;
+  ``$lib.print()`` is not required in order to use variables or methods.
+  
+  In some examples the Storm :ref:`storm-spin` command is used to suppress display of the node itself.
+  We do this for cases where displaying the node detracts from illustrating the value of the variable.
+
+In some instances we have included "use-case" examples, where the variable or method is used in a sample
+query to illustrate a possible practical use. These represent exemplar Storm queries for how a variable
+or method might be used in practice. While we have attempted to use relatively simple examples for clarity,
+some examples may leverage additional Storm features such as `subqueries`_, `subquery filters`_, or 
+`control flow`_ elements such as for loops or switch statements.
 
-In some instances we have also included "use-case" examples, where the variable or method is used in one or more sample queries to illustrate possible practical use cases. These represent exemplar Storm queries for how a variable or method might be used in practice. While we have attempted to use relatively simple examples for clarity, some examples may leverage additional Storm features such as subqueries (:ref:`storm-ref-subquery`), subquery filters (:ref:`filter-subquery`), or flow control elements such as "for" loops or "switch" statements (:ref:`storm-adv-control`).
 
 .. _meth-node:
 
 $node
 -----
 
-:ref:`vars-node-node` is a built-in Storm variable that references **the current node in the Storm query pipeline.** ``$node`` can be used as a variable on its own or with the example methods listed below. See the :ref:`stormprims-storm-node-f527` section of the :ref:`stormtypes-prim-header` technical documentation for a full list.
+:ref:`vars-node-node` is a built-in Storm variable that references **the current node in the Storm query pipeline.**
+``$node`` can be used as a variable on its own or with the example methods listed below. See the
+:ref:`stormprims-storm-node-f527` section of the :ref:`stormtypes-prim-header` technical documentation
+for a full list.
 
 .. NOTE::
 
-  As the ``$node`` variable and related methods reference the current node in the Storm pipeline, the variable and its methods will contain (and return) a null value if the inbound result set is empty (i.e., contains no nodes).
+  As the ``$node`` variable and related methods reference the current node in the Storm pipeline, the variable
+  and its methods will contain (and return) a null value if the inbound result set is empty (i.e., contains
+  no nodes).
```

Comment:
Doesn't look like you changed this but it isn't quite correct. If there aren't any nodes in the pipeline, statements which contain `$node` won't be executed at all.

Reply:
revised wording